test/parse/test_document_type_declaration.rb

```diff
@@ -53,6 +53,53 @@ def test_no_name
       end
     end
 
+    class TestUnclosed < self
+      def test_unclosed_doctype_only
```

Comment:
`unclosed_doctype_` is redundant because file name is `test_document_type_declaration.rb` and class name is `TestUnclosed`.

```suggestion
      def test_no_extra_node
```